BraintreePSP/BraintreePaymentManager.swift

```diff
@@ -8,6 +8,7 @@
 
 import Foundation
 import KarhooSDK
+import KarhooUISDK
```

Comment:
These were deleted to remove warnings of them being unnecessarily added. Could you please add a note to both files that they are required for SPM to work so that we avoid this situation in the future? Maybe even suppress the warning altogether for this specific case?

Reply:
Good idea. I've added this comment: `// Even if the import may not be required in some configurations, it is needed for SPM linking. Do not remove unless you are sure a new solution allowes SPM-based project/app to be compiled.`